### ISSUE TITLE
[PATCH] rtexif: add Lens ID2 for new Sony FE 85/1.8

### ISSUE THREAD

**Issue by mandree** [Beep6581/RawTherapee#3800]
RawTherapee (up-to-date Git dev branch) misdetects the Sony FE 85/1.8 lens (shipping since mid March 2017 only) on the Alpha 7 series cameras as though it were the Zeiss Batis 85/1.8 OSS lens instead.

Minimal patch to correct that at https://gist.github.com/mandree/f175a1411c8d0d7a747e0a55298c77e9

**Comment by mandree** [comment]
A general comment, do we really need to list all that stuff with the bogus 6553 and 65535 LensIds? Especially so that ID2 becomes effective at all?

And while we're at it, we don't we just use the LensModel or LensSpec tags?
Some cameras apparently write plain text into a tag - this is from `exiftool -s -G -H` which looks pretty reasonable to me:

```
[EXIF]          0xa432 LensInfo                        : 85mm f/1.8
[EXIF]          0xa434 LensModel                       : FE 85mm F1.8
[MakerNotes]    0xb027 LensType                        : E-Mount, T-Mount, Other Lens or no lens
[MakerNotes]    0xb02a LensSpec                        : FE 85mm F1.8
[MakerNotes]    0x001e LensZoomPosition                : 0%
[MakerNotes]    0x0008 LensMount2                      : E-mount
[MakerNotes]    0x0009 LensType3                       : Unknown (32796)
[MakerNotes]    0x000d LensE-mountVersion              : 1.60
[MakerNotes]    0x0015 LensFirmwareVersion             : Ver.01
[MakerNotes]    0x0105 LensMount                       : E-mount
[MakerNotes]    0x0106 LensFormat                      : Full-frame
[MakerNotes]    0x0107 LensType2                       : Unknown (32796)
[MakerNotes]    0x0116 LensSpecFeatures                : FE
[Composite]          - LensID                          : FE 85mm F1.8
```

**Comment by Beep6581** [comment]
Thank you for the patch. I will merge your patch once `wallfix` is merged into `dev`. I hope to find time to include #3802 in RT5.1.
